test/pbxFile.js

```diff
@@ -89,10 +89,10 @@ exports['lastKnownFileType'] = {
         test.done();
     },
 
-    'should set lastKnownFileType to unknown if undetectable': function (test) {
+    'should set lastKnownFileType to null if undetectable': function (test) {
         var sourceFile = new pbxFile('Plugins/ChildBrowser.guh');
 
-        test.equal('unknown', sourceFile.lastKnownFileType);
+        test.equal(null, sourceFile.lastKnownFileType);
```

Comment:
Looks like an unwanted change, which was taken back out by subsequent commits. If accepted we should probably do a squash merge.